src/Microsoft.AspNet.SignalR.SqlServer/install.sql

```diff
@@ -88,7 +88,7 @@ PRINT 'Current SignalR schema version: ' + CASE @CURRENT_SCHEMA_VERSION WHEN NUL
 -- Install tables, etc.
 IF @CURRENT_SCHEMA_VERSION IS NULL OR @CURRENT_SCHEMA_VERSION <= @TARGET_SCHEMA_VERSION
 	BEGIN
-		IF @CURRENT_SCHEMA_VERSION IS NULL OR @CURRENT_SCHEMA_VERSION = @TARGET_SCHEMA_VERSION
+		IF @CURRENT_SCHEMA_VERSION IS NULL OR @CURRENT_SCHEMA_VERSION < @TARGET_SCHEMA_VERSION
```

Comment:
The number of tables is a configuration parameter for Sql Server based scaleout - https://github.com/SignalR/SignalR/blob/dev/src/Microsoft.AspNet.SignalR.SqlServer/SqlScaleoutConfiguration.cs#L36